app/src/main/java/com/asfoundation/wallet/di/ToolsModule.java

```diff
@@ -834,7 +834,7 @@ AsfInAppPurchaseInteractor provideAsfInAppPurchaseInteractor(
 
   @Singleton @Provides AnalyticsAPI provideAnalyticsAPI(OkHttpClient client,
       ObjectMapper objectMapper) {
-    return new Retrofit.Builder().baseUrl("http://ws75.aptoide.com/api/7/")
+    return new Retrofit.Builder().baseUrl("https://ws75.aptoide.com/api/7/")
```

Comment:
Ticket already done in dev

Reply:
It was missing the merge with dev. Done.